doc/content/guides/getting-started/console/_index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Console"
+description: ""
+weight: 8
+---
+
+<style>
+img {
+  border: 1px solid #ddd;
+}
+</style>
+
+## Login
+
+You need to be logged in inside the Console in order to create gateways, applications, devices and API keys. With the stack running in one terminal session, open https://localhost:8885 in your browser.
+
+>Note: If you are hosting your stack on a remote machine, you should replace `https://localhost:8885` with your own host address and port.
+
+{{< figure src="front.png" alt="Front page" >}}
+
+You are now on the Console landing page. Click `Login with your TTN Stack Account` in order to reach the login page.
+
+{{< figure src="login.png" alt="Login" >}}
+
+After entering your credentials and logging in, you will reach the Console overview page. You can now fully exploit the capabilities of the Console.
+
+{{< figure src="overview.png" alt="Overview" >}}
+
+## Create gateway
+
+Click on `Register a gateway` in the overview page in order to reach the gateway registration page. Fill the gateway ID, gateway EUI, gateway name and frequency plan and click `Create Gateway`.
```

Comment:
Let's leave out the name, the CLI guide also doesn't talk about that

Reply:
Agree.
Does the console auto fill the fields without the * ? If that the case it should be precised.

Reply:
Those fields are optional, you can just leave them empty.
Anything auto-filled will already be auto-filled before the user sees the form.

Reply:
What about the EUI, isn't it needed for UDP gateway ?

Reply:
Yes for UDP and Basic Station gateways, but not for MQTT or gRPC gateways.